src/main/webapp/creatives/maps/simid_map_creative.test.js

```diff
@@ -0,0 +1,64 @@
+/** Tests to see if Find Nearest X button is appending correct text */
+
+import SimidMapCreative from './simid_map_creative.js';
+
+test('testing button given ad params', ()=> {
+    document.body.innerHTML = '<button id="findNearest"></button>';
+
+    const exampleMap = new SimidMapCreative();
```

Comment:
nit. don't use example (its not an example, its a test).  testCreative, or even just simidCreative/mapCreative etc. would be better

Reply:
changed to test